packages/core/src/components/dialog/multistepDialog.tsx

```diff
@@ -17,6 +17,8 @@
 import classNames from "classnames";
 import * as React from "react";
 
+import { Tooltip } from "@blueprintjs/core";
```

Comment:
You don't need to import from `@blueprintjs/core` -- this _is_ the source code for that package. Within this package, you can import other components in the `src/` directory, just like you see below on L24 with `Button`, etc.

This import should be `from "../tooltip/tooltip";`